### ISSUE TITLE
Force save sets read-only

### ISSUE THREAD

**Issue by jhihn** [Komodo/KomodoEdit#693]
Komodo is having  hard time doing the right thing with files that are remote. These files in this problem are "mount"ed as to appear local (not using Komodo's SCP)

The file is opened in a non-writable state. 
The VCS makes it writable (or just a chmod command).
Komodo doesn't detect the chmod
The file is edited.
The file is saved.
The Force save message appears, told to force.
The file is written, with the read-only permissions. 
I'd like it to either:
a) Ideally detect the file mode properly before save and not show the dialog
b) give an option of saving as writable

Adding to this is if I "Refresh file status", sometimes it works and detects the  file is no longer read-only, but the majority of the time, the locked icon in the tab remains and it still thinks it is locked.

As I work mainly on remote files this is a problem.

I've verified the permissions at every step with ls -l

**Comment by Naatan** [comment]
Context: http://forum.komodoide.com/t/komodo-requires-force-save-of-writable-file/1803/11

TLDR version:

 * Mark file read only
 * Open file in Komodo
 * Mark file writeable 
 * Save in Komodo
 * Confirm to force save
 * File is now read only again

**Comment by Naatan** [comment]
I am unable to reproduce this. Carey can you repro?

**Comment by Naatan** [comment]
@jhihn after marking the file as writeable does Komodo still show the file lock icon?

**Comment by jhihn** [comment]
Yes.

**Comment by Naatan** [comment]
What's your OS? (Platform and version).
